### PR TITLE
Mr user context

### PR DESCRIPTION
**Pull request by mriotortoig**

### CODE REVIEW

File: en/web/user-context.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# User Context
+
+Used to identify the user, related to a particular call to the methods of `Reveal.Sdk.IRVDashboardProvider`, `IRVAuthenticationProvider`, `IRVDataProvider` and other providers. You
```
Comment (brianlagunas) [inline en/web/user-context.md:3]: Let's make sure we are clearly explaining what the topic is about. For example:

"The User Context represents the identity of the authenticated user of the application. The User Context can be used by Reveal SDK providers such as the `IRVDashboardProvider`, `IRVAuthenticationProvider`, `IRVDataProvider` and others to restrict what permissions the user has. To provide User Context to the Reveal SDK, you must create a class that implements the `IRVUserContextProvider` interface."

File: en/web/user-context.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                userId,
+                new Dictionary<string, object>() { {"someKey", aspnetContext.Current.Request.Cookies["someCookieName"].Value } });
+        }
+    }
+```
+
+You can find an implementation in [**Web sample application using RevealBI AspNetCore SDK**](https://github.com/RevealBi/sdk-samples-aspnetcore/blob/590f79ce822755002bf2ccbbdb6e455ab7f1f3c3/Cookies-Auth/README.md)
+
+
+### API Reference Links
```
Comment (brianlagunas) [inline en/web/user-context.md:38]: Did you decide not to just use links on the word within the topic?

[IRVUserContextProvider ](https://help.revealbi.io/api/aspnet/latest/Reveal.Sdk.IRVUserContextProvider.html)

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: en/developer/java-sdk/using-the-server-sdk/user-context.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Used to identify the user, related to a particular call to the methods of `IRVDashboardProvider`, `IRVAuthenticationProvider`, `IRVDataProvider` and other providers. You can store some user specific data in the Properties like user's permission that you want to use in some of the providers listed above. You can use `RVUserContext` as a default implementation.<br>
+This way, you can perform user specific interactions within Reveal SDK.
+
+
+## Sample User Context Provider
+
+`SampleUserContextProvider` extends `RVContainerRequestAwareUserContextProvider` class and defines the `getUserContext` method that returns a `RVUserContext` object, which contains both the `userId` and a set of properties. We're using this list of properties to store the sessionId, this way we can retrieve it later when credentials for a data source are requested.
+
+```java
+public class SampleUserContextProvider extends RVContainerRequestAwareUserContextProvider {
```
Comment (brianlagunas) [inline en/developer/java-sdk/using-the-server-sdk/user-context.md:12]: I assume Java doesn't work the same way as C# so we have this really long class name to use instead and can't use an interface?

Comment (mriotortoig) [reply en/developer/java-sdk/using-the-server-sdk/user-context.md:12]: Indeed, as per public interface IRVUserContextProvider
Interface that can be implemented to return current user's context when a request is being processed. You can extend RVContainerRequestAwareUserContextProvider and implement the abstract method that receives an instance of ContainerRequestContext from JAX-RS specification.

File: en/web/user-context.md (hunk 2 above)
Comment (mriotortoig) [reply en/web/user-context.md:38]: I will rephrase

**Review by brianlagunas — APPROVED** [review]

**Pull request merged**